src/get-started/editor.md

```diff
@@ -41,92 +50,132 @@ that's OK, skip ahead to the [next step: Test drive][].
+[Install on macOS]: https://code.visualstudio.com/docs/setup/mac
+[Install on Windows]: https://code.visualstudio.com/docs/setup/windows
+[Install on Linux]: https://code.visualstudio.com/docs/setup/linux
+
+## Install the VS Code Flutter plugin
+
+1. Start **VS Code**.
+
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
+
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
+
+1. Type `install`.
+
+1. Select **Extensions: Install Extensions**.
+
+1. Type `flutter` in the extensions search field.
+
+1. Select **Flutter** in the list.
+
+1. Click **Install**.
+   Installing the Flutter plugin also installs the Dart plugin.
 
-* [VS Code][], latest stable version
+## Validate your VS Code setup with the Flutter Doctor
 
-## Install the Flutter and Dart plugins
+1. Go to **View** <span aria-label="and then">></span>
+   **Command Palette...**.
 
- 1. Start VS Code.
- 1. Invoke **View > Command Palette...**.
- 1. Type "install", and select
-    **Extensions: Install Extensions**.
- 1. Type "flutter" in the extensions search field,
-    select **Flutter** in the list, and click **Install**.
-    This also installs the required Dart plugin.
+   You can also press <kbd>Control</kbd> / <kbd>Command</kbd> +
+   <kbd>Shift</kbd> + <kbd>P</kbd>.
 
-## Validate your setup with the Flutter Doctor
+1. Type `doctor`.
 
- 1. Invoke **View > Command Palette...**.
- 1. Type "doctor", and select the
-    **Flutter: Run Flutter Doctor**.
- 1. Review the output in the **OUTPUT** pane for any issues.
-    Make sure to select Flutter from the dropdown
-    in the different Output Options.
+1. Select the **Flutter: Run Flutter Doctor**.
+
+1. Review the output in the **OUTPUT** pane for any issues.
+   Make sure to select Flutter from the dropdown
+   in the different Output Options.
 
 </div>
 <div class="tab-pane" id="androidstudio" role="tabpanel" aria-labelledby="androidstudio-tab" markdown="1">
 
-## Install Android Studio
-
-Android Studio offers a complete,
-integrated IDE experience for Flutter.
+## Install Android Studio or IntelliJ IDEA
 
-* [Android Studio][], version 2020.3.1 (Arctic Fox) or later
+Android Studio and IntelliJ IDEA offer complete,
+integrated IDE experience once you install the Flutter plugin.
 
-Alternatively, you can also use IntelliJ:
+To install of the following IDEs, follow their instructions:
 
-* [IntelliJ IDEA Community][], version 2021.2 or later
-* [IntelliJ IDEA Ultimate][], version 2021.2 or later
+- [Android Studio][], version 2020.3.1 (Arctic Fox) or later
+- [IntelliJ IDEA Community][], version 2021.2 or later
+- [IntelliJ IDEA Ultimate][], version 2021.2 or later
 
-## Install the Flutter and Dart plugins
+## Install the Flutter plugin
 
 The installation instructions vary by platform.
 
-### Mac
+### macOS
+
+Use the following instructions for macOS:
+
+1. Start Android Studio.
+
+1. Go to **App menu** <span aria-label="and then">></span>
```

Comment:
What's "App menu"? I don't see this when I open Android Studio.

Do you mean the **Android studio** in the macOS menu bar?

Reply:
**comment:** Yes. That's what Apple's macOS docs call it. Since this covers Android Studio and IntelliJ, I couldn't write one or the other.

Reply:
I think users will be more familiar with "macOS menu bar" (https://developer.apple.com/design/human-interface-guidelines/the-menu-bar).  

I think it's fine to be a bit more verbose if it will prevent confusion. Perhaps something like "Click **Android Studio** or **IntelliJ** in the macOS menu bar (and then...)"?